main/kotlin/com/example/people_managment_system/PostgresqlApplication.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.people_managment_system
```

Comment:
I recommend to avoid using underscores, hyphens or any of these characters for packages.

Also recommend following a company domain to make it a better example, maybe go with `com.ciarankieran.core.service`

`core` indicates the domain. A domain can be like payroll or something specific in a company and `service` would be a part of `core`. If it's authentication then it can be `core.authentication`.